libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/model/ManeuverStepDistanceOptions.kt

```diff
@@ -0,0 +1,79 @@
+package com.mapbox.navigation.ui.maneuver.model
+
+import androidx.annotation.StyleRes
+import androidx.core.widget.TextViewCompat
+import com.mapbox.navigation.ui.maneuver.R
+import com.mapbox.navigation.ui.maneuver.view.MapboxStepDistance
+
+/**
+ * Specifies options for styling [MapboxStepDistance].
+ *
+ * @param textAppearance change the text appearance of total step distance and distance remaining.
+ * @see [TextViewCompat.setTextAppearance]
+ */
+class ManeuverStepDistanceOptions private constructor(
```

Comment:
nit: feels like this option should be flattened into `ManeuverViewOptions`

```
ManeuverViewOptions.stepDistanceTextAppearance

ManeuverViewOptions.Builder()
  .stepDistanceTextAppearance(R...my_text_apperance)
  .build()
```

vs
```
ManeuverViewOptions.Builder()
  .stepDistanceOptions(
    ManeuverStepDistanceOptions.Builder()
      .textAppearance(R...my_text_apperance)
      .build()
  )
  .build()
```

Reply:
Yeah I am keeping them separate on purpose so that this view is not tied to `MapboxManeuverView` and can live independently, so that it could be taken out and moved anywhere else on the screen.

Reply:
like, you want to reuse `ManeuverStepDistanceOptions` somewhere? 🤔  

The name seems tied to the ManeuverView, so maybe make it to be reused? `DistanceViewOptions`

Reply:
Done.